cucumberjs/features/pageobjects/select-permit.js

```diff
@@ -132,6 +120,30 @@ class SelectPermit {
         return this.browser.wait(until.elementLocated(By.id("chosen-permit-sr2008-no-27-code")), 5 * 20000);
     }
 
+    ////////////////////////////////
+
+    async permitSR2015No18Radio() {
+        return this.browser.wait(until.elementLocated(By.id("chosen-permit-sr2015-no-18-input")), 5 * 20000);
+    }
+
+    async permitSR2015No13Radio() {
+        return this.browser.wait(until.elementLocated(By.id("chosen-permit-sr2015-no-13-input")), 5 * 20000);
+    }
+
+    async permitSR2010No5Weight() {
+        return this.browser.wait(until.elementLocated(By.id("chosen-permit-sr2010-no-4-weight")), 5 * 20000);
+    }
+
+
```

Comment:
Is there are reason for lots of blank lines?

Reply:
Sorry will be doing code tidy up where i will remove all of these